frontend-react/src/network/api/OrgApi.test.ts

```diff
@@ -0,0 +1,42 @@
+import { orgApi } from "./OrgApi";
+
+describe("Organization API", () => {
+    test("getOrgList", () => {
+        const endpoint = orgApi.getOrgList();
+        expect(endpoint).toEqual({
+            method: "GET",
+            url: `${process.env.REACT_APP_BACKEND_URL}/api/settings/organizations`,
+            headers: {
+                Authorization: "Bearer ",
+                Organization: "",
```

Comment:
I can `setStoredOktaToken` and `setStoredOrg` in a `beforeAll` and not have to leave this empty